app/styles/ui/_commit-message-avatar.scss

```diff
@@ -38,6 +38,7 @@
       height: 10px;
       // With width=100%, the icon will be centered horizontally
       width: 100%;
+      vertical-align: unset;
```

Comment:
Added this after taking windows screencast and seeing the waring icon out of place.

Before
![CleanShot 2023-03-06 at 07 29 08](https://user-images.githubusercontent.com/75402236/223110663-ce30377c-81ad-40b5-a390-93d6ac965805.png)

After: 
![CleanShot 2023-03-06 at 07 29 30](https://user-images.githubusercontent.com/75402236/223110732-54eebb21-0279-401d-bb3e-8e35eada1f6e.png)